com.minres.coredsl/src/com/minres/coredsl/type/FloatType.xtend

```diff
@@ -0,0 +1,18 @@
+package com.minres.coredsl.type
+
+import org.eclipse.xtend.lib.annotations.Data
+
+@Data
+class FloatType extends CoreDslType {
```

Comment:
We actually don't have to support FP types right now, though I suppose it doesn't hurt to prepare for them as they are planned to become part of CoreDSL at some point in the future.

Reply:
I only added that type because I needed a non-integer value type for the `InvalidIsaParameterType` test case.